### ISSUE TITLE
Does not work in mobile chrome after update to version 55.0.X

### ISSUE THREAD

**Issue by JosMarRivera** [apeatling/web-pull-to-refresh#20]
Web-pull-to-refresh does not work in chrome browser for mobile after the recent update to version 55.0.X

**Comment by pskwarko** [comment]
You just need to update hammer at the top of the js file to the v2.0.8

found here http://hammerjs.github.io/dist/hammer.min.js

Hopefully that helps!

Edit: Upon further investigation this didn't actually fix the problem, just fixed it in chrome desktop mode.

**Comment by itaischw** [comment]
Is it working for you? 
I tried downloading the hammer like you suggested.
Replaced the script in my environment and Run it on Chrome.
When it is in Desktop mode all is working. But if you go to inspect and change to device mode, choose Nexus or something you will see it is not working properly. even the demo with the same steps doesn't work. Also if you simply run it on android with updated chrome it isn't working.
What am I missing?

**Comment by itaischw** [comment]
Any thoughts?

**Comment by pskwarko** [comment]
You're right, I jumped the gun on that one. It worked in the chrome browser desktop mode, but not when you actually throw it on a real mobile device. I had to pull the library out of our app, and go with iScroll instead. Let me know if you find a better solution.

**Comment by itaischw** [comment]
Not so much...working to find a solution. What do you mean but go with iScroll you removed this completely? Do iScroll has pull to refresh too? anyway I will try to see if I can solve this issue here.
Let me know if you have any progress

**Comment by pskwarko** [comment]
I had to pull wptr from our hybrid app, and went with a custom iScroll implementation. Essentially just hand-rolled ptr in. Spent a long time searching for a solution to fix this, but no luck,
